pkgs/by-name/ai/aiken/package.nix

```diff
@@ -10,16 +10,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "aiken";
-  version = "1.0.29-alpha"; # all releases are 'alpha'
+  version = "1.1.2"; # all releases are 'alpha'
```

Comment:
Moved to https://github.com/NixOS/nixpkgs/pull/339394. I want this comment removed